src/components/content-switcher.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button } from "./ui/button";
+
+type ContentSwitcherProps = {
+  toggle: boolean;
+  setToggle: (toggle: boolean) => void;
+};
+
```

Comment:
look in the future please for some improvements and maybe better look of switcher